script.js

```diff
@@ -1,62 +1,55 @@
-const imgArr=["./num0.jpg", "./num1.png", "./num2.png", "./num3.png", "./num4.png", "./num5.png", "./num6.jpeg"
-, "./num7.png", "./num8.png", "./num9.jpg", "./num10.jpg", "./num11.png", "./num12.png", "./num13.png"
-, "./num14.png", "./num15.png", "./num16.jpeg", "./num17.png", "./num18.png", "./num19.jpg"];
-var clickedNumArr=[]; 
-var clickedIdArr=[];
-var randImgArr=[ "./num0.jpg","./num1.png", "./num2.png", "./num3.png", "./num4.png", "./num5.png", "./num6.jpeg"
-, "./num7.png", "./num8.png", "./num9.jpg","./num10.jpg", "./num11.png", "./num12.png", "./num13.png"
-, "./num14.png", "./num15.png", "./num16.jpeg", "./num17.png", "./num18.png", "./num19.jpg"];
+const imgArr = Array(20).fill().map((_, i) => `./num${i}.png`);
+
+let clickedNumArr=[];
+let clickedIdArr=[];
+
+let randImgArr = imgArr.slice();

```

Comment:
Slice creates a shallow copy of an array. You can look up "shallow copy vs deep copy" on google for more info.